sql/catalyst/benchmarks/HashByteArrayBenchmark-jdk11-results.txt

```diff
@@ -2,76 +2,76 @@
 Benchmark for MurMurHash 3 and xxHash64
 ================================================================================================
 
-OpenJDK 64-Bit Server VM 11.0.10+9-LTS on Linux 5.4.0-1043-azure
-Intel(R) Xeon(R) CPU E5-2673 v4 @ 2.30GHz
+OpenJDK 64-Bit Server VM 11.0.13+8-LTS on Linux 5.11.0-1022-azure
+Intel(R) Xeon(R) Platinum 8272CL CPU @ 2.60GHz
 Hash byte arrays with length 8:           Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
 ------------------------------------------------------------------------------------------------------------------------
-Murmur3_x86_32                                       15             16           1        140.1           7.1       1.0X
-xxHash 64-bit                                        18             19           1        118.8           8.4       0.8X
-HiveHasher                                           17             18           1        122.4           8.2       0.9X
+Murmur3_x86_32                                       13             13           0        159.2           6.3       1.0X
+xxHash 64-bit                                        12             12           0        173.6           5.8       1.1X
```

Comment:
This case looks strange because the ratio (Murmur vs xxHash) is reversed somehow.